### PR TITLE
feat: add better open sauced asset branding

### LINKED ISSUE #32: Feature: add more opensauced.pizza branding
### Type of feature

Style

### Current behavior

We only have one logo and no pizza

### Desired solution

Add more sauce to the footer, the header and, potentially, the navigation bars.

Announcement bar with a default message would be nice.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Deadreyo**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds more of the Open Sauce branding and graphical identity to the website, through images and icons.

## Related Tickets & Documents
Fixes #32 

## Mobile & Desktop Screenshots/Recordings

image
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
Nothing

### CODE REVIEW

**Comment by Deadreyo** [comment]
This failing check is not a result of my work nor is it related to it in any way. My work only touched the styling and the text content of the website.

Awaiting reviewing. @bdougie @0-vortex

File: docs/introduction.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
-# Introduction to Open Sauced
+# Introduction to Open Sauced ![img](../static/favicon.ico)
 
 ![Example banner](../static/img/what-is-opensauced.png)
 
 [Watch this on YouTube](https://www.youtube.com/watch?v=CKbTdYZAvSM&list=PLHyZ0Wz_A44XSYlBAfO2nBqoYaJJ5fdu5)
 
-Open Sauced provides guidance for new contributors finding their next contribution. Our approach towards onboarding offers a way to track contributions through a GitHub powered dashboard.
+Open Sauced provides guidance for new contributors finding their next contribution. Our approach towards onboarding offers a way to trdack contributions through a GitHub powered dashboard.
```
Comment (0-vortex) [inline docs/introduction.md:13]: typo here :P

File: docusaurus.config.js [hunk 2]
```diff
@@ -99,7 +109,7 @@ module.exports = {
           ],
         },
         {
-          title: 'More',
+          title: 'More ',
```
Comment (0-vortex) [inline docusaurus.config.js:112]: no need for space here

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Added some comments for very minor changes, looks good to me after that! 👍 

cc @bdougie what do you think?

**Comment by 0-vortex** [comment]
> This failing check is not a result of my work nor is it related to it in any way. My work only touched the styling and the text content of the website.
> 
> Awaiting reviewing. @bdougie @0-vortex

Hey don't worry about that, I noticed the action failing and silently ignored 😅

**Comment by Deadreyo** [comment]
Ok fixed

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍 I'll let @bdougie have the last word design wise, as fas as code is concerned, simple, clean and timely PR, great first contribution! 🍕

**Comment by bdougie** [comment]
I think it looks good and adds flair. Also timely. Opened two related issues.

https://github.com/open-sauced/docs.opensauced.pizza/issues/47
https://github.com/open-sauced/docs.opensauced.pizza/issues/48

**Review by bdougie — APPROVED** [review]

**Pull request merged**